djangorestframework_camel_case/util.py

```diff
@@ -16,10 +16,15 @@ def camelize(data):
             new_key = re.sub(r"[a-z]_[a-z]", underscoreToCamel, key)
             new_dict[new_key] = camelize(value)
         return new_dict
-    if isinstance(data, (list, tuple)):
+    if isinstance(data, list):
         for i in range(len(data)):
             data[i] = camelize(data[i])
         return data
+    if isinstance(data, tuple):
+        camelized_data = []
+        for i in range(len(data)):
```

Comment:
You are absolutely correct. But this is obsolete since PR #18 as well as this repository seems to be abandoned I won't change it, unless of course you can and want to merge this PR.